js/hoverzoom.js

```diff
@@ -140,6 +140,9 @@ var hoverZoom = {
                     if (hzCaption.height() > 20) {
                         hzCaption.css('font-weight', 'normal');
                     }
+                    if(options.enableDarkMode) {
```

Comment:
please respect existing code style - have a single space on both sides of parentheses.